secure-biometric-device/utility/sbiservice.js

```diff
@@ -0,0 +1,297 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import axios from "axios";
+import { localStorageService } from "./localstorage";
+import * as jose from "jose";
+
+const {
+  addDeviceInfos,
+  addDiscoveredDevices,
+  clearDeviceInfos,
+  clearDiscoveredDevices,
+} = {
+  ...localStorageService,
+};
+const purpose = "Auth";
+const certification = "L1";
+
+const DeviceStatusReady = "Ready";
+
+/**
+ * endpoint for the biometric device
+ */
+const deviceEndPoint = "/device";
+const infoEndPoint = "/info";
+const captureEndPoint = "/capture";
+
+/**
+ * http method for retrieving from &
+ * sending data to the biometric devices
+ */
+const mosip_DiscoverMethod = "MOSIPDISC";
+const mosip_DeviceInfoMethod = "MOSIPDINFO";
+const mosip_CaptureMethod = "CAPTURE";
+
+const BioType = {
+  FACE: "Face",
+  FINGER: "Finger",
+  IRIS: "Iris",
+};
+
+class SbiService {
+  esignetConfig;
```

Comment:
this is a helper library to sbi spec, esignet naming or reference should not be used.